packages/shell-api/src/helpers.ts

```diff
@@ -156,8 +161,10 @@ export async function getPrintableShardStatus(db: Database, verbose: boolean): P
 
   const version = await versionColl.findOne();
   if (version === null) {
-    throw new MongoshInvalidInputError('This db does not have sharding enabled.' +
-      ' be sure you are connecting to a mongos from the shell and not to a mongod.');
+    throw new MongoshInvalidInputError(
+      'This db does not have sharding enabled. Be sure you are connecting to a mongos from the shell and not to a mongod.',
+      ShellApiErrors.NoSharding
```

Comment:
It has to be connected to mongos and sharded, so I think better this error:
```suggestion
      ShellApiErrors.NotConnectedToMongos
```